docs/units/format.rst

```diff
@@ -221,10 +225,71 @@ this behavior:
   - ``'silent'``: return an `~astropy.units.UnrecognizedUnit`
     instance.
 
+By adding additional unit aliases  for the misspelt units with
+`~astropy.units.set_enabled_aliases` (e.g. 'Angstroms' for 'Angstrom'), and new units
+via `~astropy.units.def_unit` and `~astropy.units.add_enabled_units`, we can use
+``parse_strict='raise'`` to rapidly find issues with the units used, while also
+being able to read in older datasets where the unit usage may have been less
+standard.
+
+.. tip::
+
+    Not all formats currently support defining aliases. If you are using a
+    unit format that is not supported, you can convert the unit string back
+    into a Python string, and reparse the string with the default format::
```

Comment:
Hm, I found the phrasing here a little confusing.

Reply:
Fortunately, I found a different method that means all formats are now supported, so this can go. Will push a revised PR shortly.